### DISCUSSION TITLE
DNS not working on US West servers

### DISCUSSION THREAD

**Discussion by arslnb** [vercel/community#434] [Help] [answered]
Hi team, 

Our domain whose DNS is managed by Vercel is currently down in some locations. It's hard to debug and is a production issue. 

We're pointing livedocs.com to 76.76.21.21 (Vercels IP) and https://cachecheck.opendns.com/ reports a SERVFAIL on some locations.

**Comment by swarnava** [comment, marked as answer]
Hi Arsalan,

I have noticed you are already in touch with our team to debug this. We are investigating this and keep you updated via the email thread. Thanks!
